schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int,
```

Comment:
- [ ] [OPTIONAL] I think it would be better to make the `id column` not null and `autoincremented` and also to mark it as primary key 👍 

ex: 
```
    id int NOT NULL IDENTITY(1,1)
    PRIMARY KEY (id)
```
 _An auto-incremented primary key ensures that each row in the table has a unique identifier. This is useful for a number of reasons, including data integrity and ease of retrieval 😂 ._